packages/client/src/pages/sign-in.tsx

```diff
@@ -1,31 +1,62 @@
 import React from "react";
-
+import * as Yup from "yup";
+import { useRouter } from "next/router";
+import { useLoginMutation } from "../generated/codegen";
 import {
   Formik,
   Center,
   Button,
   Form,
   InputField,
   Heading,
+  useToast,
 } from "@mustardmind/mauinz";
+import { toErrorMap } from "src/util/toErrorMap";
 
-interface SignInFormValues {
-  email: string;
-  password: string;
-}
+const SignInValidationSchema = Yup.object().shape({
+  email: Yup.string().email("Invalid email").required("Required"),
+  password: Yup.string().min(8, "Too Short!").required("Required"),
+});
 
 const SignIn: React.FC<{}> = () => {
-  const initialValues: SignInFormValues = { email: "", password: "" };
+  const router = useRouter();
+  const toast = useToast();
+  const [login] = useLoginMutation();
+
   return (
     <Center flexDirection="column" h="100vh">
       <Heading>Sign In</Heading>
       <Formik
-        initialValues={initialValues}
-        onSubmit={(values, actions) => {
-          setTimeout(() => {
-            alert(JSON.stringify(values, null, 2));
-            actions.setSubmitting(false);
-          }, 1000);
+        initialValues={{ email: "", password: "" }}
```

Comment:
Add type-safety to initalvalues using `CredentialsInput` from the generated types.